plugins/catalog-backend/src/next/Stitcher.ts

```diff
@@ -137,7 +142,15 @@ export class Stitcher {
             ['backstage.io/orphan']: 'true',
           };
         }
-
+        if (errors !== '') {
```

Comment:
```suggestion
        if (errors.length) {
```

Reply:
made it even more carefully check that it's an array with actual elements